### ISSUE TITLE
Enhanctment Glint should be rendered using a custom shader rather than with additional geometry

### ISSUE THREAD

**Issue by jellysquid3** [jellysquid3/Sodium#2497]
This would solve a lot of problems with code paths getting de-optimized when it encounters the `VertexConsumer` that is used for enchantment glints. Additionally, not generating additional geometry on the CPU saves significant time and video memory bandwidth, while fixing issues like #2487 and #2486.

**Comment by muzikbike** [comment]
Could a similar method be used for the block cracking animation if not already?

**Comment by djmrFunnyMan** [comment]
Yeah block breaking animation is subject to similar issues #2454 #2447
So I think a similar fix should be considered
